src/Test/Perf/infra/run_and_report.csx

```diff
@@ -19,8 +19,8 @@ if (result)
     var elapsedTimeViBenchJsonFilePath = GetViBenchJsonFromCsv(elapsedTimeCsvFilePath, null, null);
     string jsonFileName = Path.GetFileName(elapsedTimeViBenchJsonFilePath);
 
-    Log("Copy the json file to the ViBench share");
     // Move the json file to a file-share
+    Log("Copy the json file to the ViBench share");
     File.Copy(elapsedTimeViBenchJsonFilePath, $@"\\vcbench-srv4\benchview\uploads\vibench\{jsonFileName}");
```

Comment:
I don't think that we are actually allowed to use the names of internal shares for security reasons.

This is why on some comments we use the phrase "pick up the file at a well-known location"